e2e/common/traits/health_test.go

```diff
@@ -35,7 +36,18 @@ import (
 	v1 "github.com/apache/camel-k/pkg/apis/camel/v1"
 )
 
+/*
+ * TODO
+ * Test needs to be modified as route for unsecured http is not created on OCP.
```

Comment:
It seems the comment is taken from the route trait tests. This test do not create a Route.